google/cloud/forseti/scanner/audit/firewall_rules_engine.py

```diff
@@ -613,7 +613,13 @@ def is_whitelist_violation(rules, policy):
     Returns:
       bool: If the policy is a subset of one of the allowed rules or not.
     """
-    return not any([policy < rule for rule in rules])
+    policy_subset_check = []
+    for rule in rules:
+        if policy < rule:
+            policy_subset_check.append(True)
+        else:
+            policy_subset_check.append(False)
+    return not any(policy_subset_check)
```

Comment:
Now that I'm looking at this, can we do like below, so that we can put a break point on result?
```
result = not any(policy_subset_check)
return result
```